libcni/api.go

```diff
@@ -24,10 +24,11 @@ import (
 )
 
 type RuntimeConf struct {
-	ContainerID string
-	NetNS       string
-	IfName      string
-	Args        [][2]string
+	ContainerID   string
+	NetNS         string
+	IfName        string
+	Args          [][2]string
+	RuntimeConfig map[string]interface{}
```

Comment:
This is an unfortunate naming almost-collision, but I suppose it can't be avoided.